### PR TITLE
make sure planter entrypoint is copied correctly

### PR DESCRIPTION
**Pull request by BenTheElder**
I think this should fix https://github.com/kubernetes/test-infra/issues/7779

### CODE REVIEW

File: planter/Dockerfile [hunk 1]
```diff
@@ -61,4 +61,6 @@ RUN INSTALLER="bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh"; \
 RUN touch /etc/passwd && chmod a+rw /etc/passwd
 
 COPY ./entrypoint.sh /
+RUN chmod +x ./entrypoint.sh
```
Comment (ixdy) [inline planter/Dockerfile]: I think you need `+rx`.

**Comment by BenTheElder** [comment]
/hold
verifying that this works

File: planter/Dockerfile (hunk 1 above)
Comment (BenTheElder) [reply planter/Dockerfile]:
```
chmod: cannot access './entrypoint.sh': No such file or directory
The command '/bin/bash -c chmod +x ./entrypoint.sh' returned a non-zero code: 1
```
ah, the real culprit is that copy command

**Pull request merged**